webpack.mix.js

```diff
@@ -12,4 +12,5 @@ let mix = require('laravel-mix');
  */
 
 mix.react('resources/js/app.js', 'public/js')
-  .sass('resources/sass/app.scss', 'public/css');
+  .sass('resources/sass/app.scss', 'public/css')
+  .version();
```

Comment:
Worth adding a node process env check to make sure we're building for production? Would potentially save a small amount of time each build as it won't have to hash the contents